config_defaults.yml

```diff
@@ -9,7 +9,14 @@ Gemfile:
   - gem: rspec-puppet-facts
     version: '>= 1.7'
   - gem: puppetlabs_spec_helper
-    version: '>= 0.8.0'
+    version: ">= 1.1.0', '<= 2.1.0"
+  - gem: parallel_tests
+    options:
+      platforms:
```

Comment:
do you think we still need to care about ruby1.9 (or even 2.0, 2.1)? They are all EOL since a long time.